packages/hw-common/src/views/AccountSelect.svelte

```diff
@@ -291,9 +291,12 @@
   }
 
   .hardware-connect-modal {
-    width: 50rem;
+    width: 100%;
+    flex-flow: column;
+    display: flex;
+
     max-height: 51.75rem;
```

Comment:
Do we need this max height with the new one below?

Reply:
@Adamj1232 which one below?

Reply:
Ahh my bad! I didnt see the collapsed line in the GH diff, all good!!